css/tabstrip.css

```diff
@@ -56,8 +57,19 @@
   transform: scale(0.8);
   transition: transform 150ms ease;
   width: 100px;
+  overflow: hidden;
+
 }
 
+.tab-thumbnail .image {
+  position: absolute;
+  width: inherit;
+  height: inherit;
+  -moz-force-broken-image-icon: 0;
```

Comment:
It is 0 by default. If you see the broken image icon, we need to set `browser.display.show_image_placeholders` to false in graphene.